How-to-push-to-nuget.txt

```diff
@@ -3,7 +3,7 @@
  - Run: 
         dotnet clean
         dotnet pack -c release --include-symbols --include-source
-        dotnet nuget push -s https://www.nuget.org/api/v2/package .\bin\Release\<package>.<version>.nupkg 
+        dotnet nuget push -s https://www.nuget.org/api/v2/package .\bin\Release\ProgressOnderwijsUtils.<version>.nupkg 

```

Comment:
random aside: wat is dit ook belachelijk omslachtig allemaal voor zo'n kernfeature van een packagemanager.  Ach ja.